internal/typst/typst.go

```
@@ -135,7 +135,9 @@ func (c *compiler) Compile(opts CompileOpts) (string, error) {
 
 	c.logger.Debug(context.Background(), "Executing command to compile typst document", "binary", c.binaryPath, "args", args)
 
-	cmd := exec.Command(c.binaryPath, args...)
+	// binaryPath is configured, not user input; args are separate argv elements,
+	// not shell-interpreted, so neither can inject a command.
+	cmd := exec.Command(c.binaryPath, args...) // #nosec G204 -- fixed binary, argv not shell // nosemgrep: go.lang.security.audit.dangerous-exec-command.dangerous-exec-command
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)